### ISSUE TITLE
Parcel 1.5.1 (and 1.5.0) won't run

### ISSUE THREAD

**Issue by hybridvision** [parcel-bundler/parcel#715]
Hello, this a 🐛 bug report:

I asked about this problem on Slack and @davidnagli asked me to post an issue here after we did some initial debugging with no success.

### 🎛 Configuration (.babelrc, package.json, cli command)

I have the most basic setup as shown in the getting started docs (index.html, index.js and a package.json with default values).

### 🤔 Expected Behavior

When I run `parcel index.html` the build server should start and Parcel should process my index.html

### 😯 Current Behavior

Instead, nothing seems to happen and I'm returned to my command prompt:

![image](https://user-images.githubusercontent.com/11492432/35638205-c7c1848c-06b6-11e8-94d9-083af2d38dce.png)

I can't find any errors or sign of output anywhere so I'm not sure what is causing this.

### 💁 Possible Solution

The only thing that has made Parcel work for me was downgrading to version 1.4.1 (I tried 1.5.0) but it does the same thing as 1.5.1. 

### 🌍 Your Environment

My normal shell is [Fish shell](http://fishshell.com/) running in [Hyper](https://hyper.is) but I tried with plain bash in the standard Mac terminal app and got the exact same results.

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.5.1
| Node             | 6.4.0
| npm/Yarn         | NPM 3.10.3 / Yarn 1.3.2
| Operating System | Mac OS X 10.11.6

**Comment by marek-hanzal** [comment]
![image](https://user-images.githubusercontent.com/13855126/35643955-0cc4314a-06c8-11e8-8cb3-78b52f72c266.png)

Same for me - using 1.50 and 1.5.1 leads to this error.

**Comment by davidnagli** [comment]
@marek-hanzal Maybe I’m missing something, but your issue seems **completely** different 😬

**Comment by marek-hanzal** [comment]
@davidnagli I suppose you're right; I've posted that just because this one was closest to "not working" :) - OP has a bit different (no) output, I've got at least some error so the idea far away was "it's somehome related through huge rainbow with unicorns?" If I suppress errors, I also have "no output" :).

Sorry if I'm spamming :)

**Comment by hybridvision** [comment]
Update from my end: I tried on another machine (OS X 10.9.5) and the latest version of Parcel works as expected so I'm guessing there is some kind of problem, possibly Node related, with my other system.

On this machine, I have Node 8.3.0 and both it and yarn were installed via homebrew, unlike the other machine. When I get back to the original machine, I'll probably try upgrading node and maybe use homebrew for managing it.

@marek-hanzal - I think your issue is a bit different to mine because the `parcel` binary didn't even seem to do anything for me. In version 1.5.1, do you get errors when you try to run the most basic getting started example as shown in the docs?

**Comment by Grafikart** [comment]
Stupid question (but got a problem linked to this), but do you have a package.json at the root of your directory ?

**Comment by hybridvision** [comment]
@Grafikart - yes, I had a very basic package.json (generated via `yarn init -y`). The behaviour wasn't changing even if I ran `parcel` with no files, so it seems to be a deeper problem with the parcel executable or how it was installed for these last 2 versions.

I have NVM but it has never a totally smooth operation for me. Today I updated Node to the latest version via NVM and using the newer Node when I installed Parcel, v1.5.1 finally works :tada: 

This ticket can probably be closed but I don't know if there is some kind of breaking change that was introduced with v1.5.0. Maybe it just doesn't play well with the older version of Node or NVM. Whatever the case, I'm glad to have it working!
